vendor/libraries/usabilitydynamics/lib-wp-list-table/lib/classes/class-wp-list-table.php

```diff
@@ -537,7 +537,7 @@ protected function parse_query( $query ){
                   $_query[ 'meta_query' ] = array();
                 }
 
-                if( is_array( $q['value'] ) && !in_array( $map['compare'], array( 'NOT IN', 'IN' ) ) ) {
+                if( is_array( $q['value'] ) && !in_array( $map['compare'], array( 'NOT IN', 'IN' ) ) && 'BETWEEN' != $map['compare'] ) {
                   $map['compare'] = 'IN';
```

Comment:
Yes. I am aware that this will require change that plugin. I have made changes to this plugin just to check if my code works or not. I will make changes to that plugin if you suggest me to do so.